modules/cli/pkg/cmd/runsSubmit_test.go

```diff
@@ -602,6 +602,53 @@ func TestRunsSubmitTraceFlagReturnsOk(t *testing.T) {
 	assert.Equal(t, cmd.Values().(*utils.RunsSubmitCmdValues).Trace, true)
 }
 
+func TestRunsSubmitUserFlagReturnsOk(t *testing.T) {
+	// Given...
+	factory := utils.NewMockFactory()
+	commandCollection, cmd := setupTestCommandCollection(COMMAND_NAME_RUNS_SUBMIT, factory, t)
+
+	var args []string = []string{"runs", "submit", "--user", "mytestuser"}
+
+	// When...
+	err := commandCollection.Execute(args)
+
+	// Then...
+	assert.Nil(t, err)
+
+	// Check what the user saw is reasonable.
+	checkOutput("", "", factory, t)
+
+	assert.Contains(t, cmd.Values().(*utils.RunsSubmitCmdValues).User, "mytestuser")
+}
+
+func TestRunsSubmitUserFlagMultipleUsersReturnsError(t *testing.T) {
+	// Given...
+	factory := utils.NewMockFactory()
+	commandCollection, _ := setupTestCommandCollection(COMMAND_NAME_RUNS_SUBMIT, factory, t)
+
+	var args []string = []string{"runs", "submit", "--user", "mytestuser", "anothertestuser"}
+
+	// When...
+	err := commandCollection.Execute(args)
+
+	// Then...
+	assert.NotNil(t, err)
```

Comment:
We could also check what the error message contains to make sure we got the right error back